hours/tests/test_resource_periods_as_text.py

```diff
@@ -586,6 +628,97 @@ def test_date_period_as_text_with_time_span_resource_state_no_opening_hours(reso
     )
 
 
+@pytest.mark.django_db
+def test_resource_date_periods_as_text_ordering(resource):
+    """Test that the DatePeriods are ordered correctly in the cotent."""
+    order = [
```

Comment:
Rename this to something more descriptive, e.g.
```suggestion
    date_periods = [
```
or `date_periods_data` or whatever.